assets/models/system/automl-text-classification/description.md

```diff
@@ -0,0 +1,64 @@
+Automated Machine Learning, or AutoML, is a process that automates the repetitive and time-consuming tasks involved in developing machine learning models. This helps data scientists, analysts, and developers to create models more efficiently and with higher quality, resulting in increased productivity and scalability.
+
+AutoML Text Classification enables you to classify or categorize texts into predefined groups. Your dataset should be a labeled set of texts with their relevant tags that categorize each piece of text into a predefined group.
+
+With this functionality, you can:
+* Directly use datasets coming from [Azure Machine Learning data labeling](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-text-labeling-projects?view=azureml-api-2)
+* Utilize labeled data to create NLP models without any training code.
+* Enhance model performance by selecting the appropriate algorithm and fine-tuning the hyperparameters selecting the appropriate algorithm from a [large selection of models](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=cli#supported-model-algorithms) or let AutoML find the best model for you.
+* Either download or deploy the resulting model as a endpoint in Azure Machine Learning.
+* Scale the operationalization process with the help of Azure Machine Learning's MLOps and ML Pipelines capabilities.
+
+See [How to train nlp models](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=cli) for more information.
```

Comment:
please remove en-us from all links... it sometimes won't open in other regions otherwise...

Reply:
thanks for letting me know!